js/sykmeldinger/data/sm-sykmeldinger/smSykmeldingerSagas.js

```diff
@@ -34,15 +40,40 @@ export function* hentSmSykmeldingerHvisIkkeHentet() {
     }
 }
 
+export function* bekreftSmSykmeldingLestSaga(action) {
+    const toggle = yield select(toggleNyttSykmeldingsmottak);
+    const skalBekrefte = yield select(skalBekrefteSmSykmeldingSelector);
+    if (toggle && skalBekrefte) {
+        yield put(bekrefterLestSmSykmelding());
+        try {
+            yield call(post, `${hentSyfoApiUrl(API_NAVN.SYFOSMREGISTER)}/v1/sykmeldinger/${action.smSykmelding.id}/bekreftAvvist`);
+            yield put(smSykmeldingBekreftetLest(action.smSykmelding));
+            browserHistory.push('/sykefravaer');
+            yield delay(10000);
```

Comment:
Ser ut som at det er kommet en egen knapp for å bekrefte lest. Skal det enda være en slik delay her?

Reply:
Yes, denne delay gjør at kvittering vises i 10 sek før den skjules.